modules/material/material.service.js

```diff
@@ -29,14 +29,14 @@ class MaterialsService {
   }
 
   async checkMaterialExist(data) {
-    const material = await Material.find({
+    const material = await Material.countDocuments({
```

Comment:
Please, call variable properly, because it is not an actual entity, it is a count